### ISSUE TITLE
Failure to properly assign member

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/topcoder-payment-tool#23]
I've been seeing an issue with the payment tool - the payment gets created, but the assignee doesn't get applied, so I have to go into OR, manually assign the member, and then close the payment out manually in TC Direct.  I captured the log below:

![Screen_Shot_2020-08-17_at_08_33_16](https://user-images.githubusercontent.com/21790/90345794-2bd73980-e067-11ea-8cde-28e9eec3739a.png)

I don't know *why* it's failing to register the member. I'm assuming there's more info in a log somewhere.

**Comment by birdofpreyru** [comment]
I usually create payments via Direct, and recently getting a similar experience there: the task assignee does not become the submitter, thus I have to add him via OR, then close the payment back in Direct. Thus, I guess, not the payment tool issue _per se_.

**Comment by rootelement** [comment]
Yeah, the payment tool is using the v4 direct api, so it's probably the same issue in both places. This will be refactored to v5 shortly.
